components/arm/fake/fake.go

```diff
@@ -24,6 +24,16 @@ import (
 	"go.viam.com/rdk/spatialmath"
 )
 
+// errModelName is the returned string when we want to use a path to
+// instantiate a model.
+const errModelName = "fake arm cannot be created, unsupported arm_model: "
```

Comment:
if you are going to go the direction of named errors I have a slight preference for something like `errArmModelUnsupported` since it says what about the ModelName is bad